projects/isy-angular-widgets-demo/src/app/feature/objekt-anzeigen/objekt-anzeigen.component.html

```diff
@@ -9,7 +9,11 @@
             label="{{ 'isyAngularWidgetsDemo.labels.vorname' | translate }}"
             fieldId="first-name"
             [control]="personalInfoForm.controls.firstName | formControl"
-            [validationMessages]="{required: 'isyAngularWidgetsDemo.messages.requiredMessage' | translate}"
+            [validationMessages]="
+              personalInfoForm.controls.firstName.errors
+                ? {required: 'isyAngularWidgetsDemo.messages.requiredMessage' | translate}
+                : {}
+            "
```

Comment:
Der erwartete Inputtype stimmt nicht mehr mit der Eingabe überein, bisher:
  @Input() validationMessages: {[key: string]: string} = {};
  
  Fehler: Type {required: any} | {} is not assignable to type {[p: string]: string}